Exiled.API/Features/Scp173.cs

```diff
@@ -17,6 +17,6 @@ public static class Scp173
         /// <summary>
         /// Gets a list of player ids who will be turned away from SCP-173.
         /// </summary>
-        public static HashSet<Player> TurnedPlayers { get; } = new(20);
+        public static HashSet<Player> TurnedPlayers { get; } = new(32);
```

Comment:
Why? :) 

I am not against this change, but it almost doesn't make sense

Reply:
![image](https://user-images.githubusercontent.com/24619207/174080805-f4ec5531-57ea-4722-9bfa-29df2e109a83.png)
![image](https://user-images.githubusercontent.com/24619207/174080826-650ceeac-283e-4c1b-b10d-6e839cca8c9a.png)

I don't know if there is a fundamental difference but, I don't see why we even have it set to a value. I guess to just optimize but is there a fundamental difference under the covers? If not, I'll put it back to 20 but I only upped it out of pure curiosity. I don't mind bringing it down, it _should not_ cause a difference I would hope

Reply:
Since you've already changed it, leave it as it is.